lib/linguist/languages.yml

```diff
@@ -5410,8 +5410,7 @@ Protocol Buffer Text Format:
   - protobuf text format
   extensions:
   - ".textproto"
-  - ".pbt"
-  - ".pbtxt"
+  - ".txtpb"
```

Comment:
We never remove support for an extension (or language) as files will always exist. Please leave these and if the new extension is to be the primary extension, place it first.

Reply:
Okay, makes perfect sense to me. 

What about the `pbt` extension? I see it [elsewhere in the file](https://github.com/github-linguist/linguist/blob/017c5eaf05b6536799c5fdad4457cac127c67ee5/lib/linguist/languages.yml#L5307) for PowerBuilder

Reply:
Don’t worry about it. Anything already supported should already be correctly handled… we assume things work until someone points out a problem 😁